main/exe/src/euCliReader.cxx

```diff
@@ -34,6 +34,25 @@ int main(int /*argc*/, const char **argv) {
   uint32_t timestamph_v = timestamph.Value();
   bool not_all_zero = eventl_v||eventh_v||triggerl_v||triggerh_v||timestampl_v||timestamph_v;
 
+
+  // load configuration
+    // empty configuration object, prevents crash if no config file given
+  eudaq::Configuration eu_cfg( "", "" );
```

Comment:
Couldn't you directly make this a `shared_ptr` instead of copying the thing later on?

```cpp
auto eu_cfg = std::make_shared<eudaq::Configuration>("","");
eu_cfg->Load(/*...*/);
```

Reply:
according to you comment below this is fine